include/sdf/Collision.hh

```diff
@@ -119,6 +135,15 @@ namespace sdf
     /// \return SemanticPose object for this link.
     public: sdf::SemanticPose SemanticPose() const;
 
+    /// \brief Calculate and return the MassMatrix for the collision
+    /// \param[in] _config Custom parser configuration
+    /// \param[out] _inertial An inertial object which will be set with the
```

Comment:
```suggestion
    /// \param[out] _inertial An inertial object which will be set with the
    /// \param[in] _config Custom parser configuration
```

Reply:
Corrected in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)